BookB-Pset_GeoScienceFeatureTypeGeologicalUnit (#625)
bSI-InfraRoom/IFC-Specification: IFC4x3/Properties/r/Rank_2nKOcVn1PD5Arvkf8nGOWd/Documentation.md

> Tthe rank of GeologicUnit (as defined by ISC. e.g., group, formation, member).

Reviewer: ```suggestion
>NOTE Definition according to ISC: The rank of GeologicUnit e.g., group, formation, member.
```
